notifications.json

```diff
@@ -0,0 +1,478 @@
+                        "type": "integer"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Content": {
+            "title": "Content",
+            "type": "object",
+            "properties": {
+                "title": {
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 255
+                },
+                "body_text": {
+                    "type": "string",
+                    "faker": "lorem.paragraph"
+                },
+                "body_html": {
+                    "type": "string",
+                    "faker": "lorem.paragraph"
+                }
+            },
+            "required": [
+                "title"
+            ]
+        },
+        "MessageDefinition": {
+            "title": "MessageDefinition",
+            "type": "object",
+            "properties": {
+                "to": {
+                    "description": "One fiscal code (CF).\n",
+                    "type": "string"
+                },
+                "dry_run": {
+                    "type": "boolean",
+                    "description": "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+                },
+                "time_to_live": {
+                    "type": "integer",
+                    "description": "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 4 weeks, and the default value is 4 weeks.",
+                    "minimum": 0
+                },
+                "tags": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "content": {
+                    "$ref": "#/definitions/Content"
+                }
+            },
+            "required": [
+                "to",
+                "content"
+            ]
+        },
+        "MessageStatus": {
+            "title": "MessageStatus",
+            "type": "object",
+            "properties": {
+                "status": {
+                    "type": "string",
+                    "enum": [
+                        "QUEUED",
+                        "SENDING",
+                        "SENT",
+                        "DELIVERED",
+                        "READ",
+                        "FAILED"
+                    ],
+                    "description": "QUEUED: the message has been accepeted for delivery\n\nSENDING: the message is in charge of the delivery service (ie. sms gatewy or mail server)\n\nSENT: the message was sent by the delivery service\n\nDELIVERED: the message reached the user inbox (ie. mail server or phone)\n\nREAD: the user read the message\n\nFAILED: something wong happened, check the error code and message\n\n"
```

Comment:
not all channels support all states, this may be confusing and should be throughly explained

I would limit the states to 

`QUEUED`: accepted for delivery

`DELIVERED`: delivered to the channel

`FAILED`: delivery failed

`TIMED_OUT`: in case a TTL was set